Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```
@@ -283,6 +283,37 @@
   components:
   - type: Unremoveable
 
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsGlovesNorthStar
+  name: gloves of the north star
+  description: These gloves allow you to punch incredibly fast
+  components:
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/Northstar.rsi
```

```suggestion
    sprite: Clothing/Hands/Gloves/northstar.rsi
```

This is case sensitive